Library/Homebrew/utils/install.rb

```diff
@@ -0,0 +1,41 @@
+# typed: false
+# frozen_string_literal: true
+
+require "utils/topological_hash"
+
+module Utils
+  # Helper module for installing formulae/casks.
+  #
+  # @api private
+  module Install
+    module_function
+
+    def graph_dependencies(cask_or_formula, acc = nil)
+      acc = TopologicalHash.new if acc.nil?
+      return acc if acc.key?(cask_or_formula)
+
+      if cask_or_formula.is_a?(Cask::Cask)
+        formula_deps = cask_or_formula.depends_on.formula.map { |f| Formula[f] }
```

Comment:
```suggestion
        formula_deps = cask_or_formula.depends_on
                                      .formula
                                      .map { |f| Formula[f] }
```

and same below for other chained methods